wordpress-mobile/WordPress-Android
# Feature/15359 domains update copy images styles

Fixes #15359 

<image width=320 src="https://user-images.githubusercontent.com/990349/137677383-ad7f249f-9377-443c-ac3d-840f065ee335.png" />

To test:

- Enable AppSettings -> Debug settings -> SiteDomainsFeatureConfig ✅
- Return to My Site, Notice there's a new item under Configuration -> 🌐 Domains
- Click on Domains, takes you to Domains dashboard screen
   - Check the font styles match designs
   - Check Get / Claim your domain card has top image

- Click on search for a domain button
   - Check that the top level domain is in bold


## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
